validator/sawtooth_validator/execution/context_manager.py

```diff
@@ -320,6 +320,8 @@ def __init__(self, database, state_delta_store):
 
         self._address_regex = re.compile('^[0-9a-f]{70}$')
 
+        self._namespace_regex = re.compile('^[0-9a-f]{0,70}$')
```

Comment:
A full address is 70 hex characters which represents 35 bytes. Each byte is an address path element in the radix addressing scheme. So, a given non-leaf node in the tree can have up to 2^8 pointers to children. I have always thought of a namespace as a proper sub-tree, but maybe this is unnecessary?